src/components/Layout/LayoutColumns.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { useLayout, GU } from '@aragon/ui'
+
+function LayoutColumns({ primary, secondary }) {
+  const { layoutName } = useLayout()
+  const oneColumn = layoutName === 'small' || layoutName === 'medium'
```

Comment:
I know I did it like this the other time. But I just realised that here we can also use useViewport hook and do it this way:  `const oneColumn = below('medium')`. Both are fine :) whatever you prefer.

Reply:
That's definitely nicer, In the future i'd like to useViewport for all of these types of things, the problem is it's not set-up with our breakpoints yet and a lot of the UI kit still relies on useLayout. I was thinking we should continue to useLayout for the time being and switch it all over together :)